share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -30,32 +61,32 @@ macro(rocm_create_package)
         set( CPACK_SET_DESTDIR ON CACHE BOOL "Boolean toggle to make CPack use DESTDIR mechanism when packaging" )
     endif()
 
-    if (EXISTS "/etc/os-release")
-        rocm_set_os_id(_os_id)
-        rocm_read_os_release(_version_id "VERSION_ID")
-
```

Comment:
@eidenyoshida Please review the impact of this change on math-ci. Our CI system may break if no alternate is provide to distiguish packages by OS.